src/backend/utils/misc/etc/gpcheck.cnf

```diff
@@ -54,5 +54,60 @@ dfs.namenode.handler.count = 60
 ipc.server.handler.queue.size = 3300
 dfs.datanode.handler.count = 60
 ipc.client.connection.maxidletime = 3600000
-dfs.namenode.accesstime.precision = -1
+dfs.namenode.accesstime.precision = 0
+dfs.client.read.shortcircuit = true
 
+[hdfs.non]
+hdfs.non = none
+dfs.block.access.token.enable = FALSE
+
+[hdfs.ha]
+hdfs.ha = none
+dfs.block.access.token.enable = FALSE
+
+[hdfs.kerberos]
+hdfs.kerberos = none
+dfs.block.access.token.enable = TRUE
+dfs.datanode.data.dir.perm = 750
```

Comment:
Since 700 is default values and user might have something like '755', we do not check for non-kerberos mode.